Add install instructions to README.md (#54)
jsonnet-bundler/jsonnet-bundler: README.md

## Install

```
GO111MODULE="on" go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
```

> **NOTE**: please use the latest go to do this, ideally go 1.13 or greater.

Reviewer: ```suggestion
**NOTE**: please use a recent Go version to do this, ideally Go 1.13 or greater.
```

This will put `jb` in `$(go env GOPATH)/bin`. If you encounter the error

> `jb: command not found` after installation then you may need to either add that directory to your `$PATH` as shown [here](https://golang.org/doc/code.html#GOPATH)

Reviewer: ```suggestion
`jb: command not found` after installation then you may need to add the Go binary directory to your `$PATH` as shown [in their docs](https://golang.org/doc/code.html#GOPATH).
```

Using "here" in links is hard for screen-readers, using a descriptive name is better :D
Furthermore, looks like the `either` is misplaced there :)
